default.json

```diff
@@ -232,6 +343,18 @@
       ],
       "allowedVersions": "<1.26.0",
       "description": "kubectl is supported within one minor version (older or newer) of kube-apiserver"
+    },
+    {
+      "matchPackagePatterns": [
+        "moby/moby"
+      ],
+      "allowedVersions": "<21.0.0"
+    },
+    {
+      "matchPackagePatterns": [
+        "etcd-io/etcd"
+      ],
+      "allowedVersions": "<3.5.0"
```

Comment:
Not sure whether this is the correct approach for `regexmanagers`. Any thoughts on this @superseb @macedogm?

Reply:
This is fine but do we want these restrictions globally? Normally we include the matcher in global config and include specific restrictions in each repository (like `rancher/shell`). Unless we have "an agreement" that we all use this restriction (like Golang 1.20)

Reply:
Removing those global restrictions, as I agree this is better place at the individual projects.